src/dotnet/CarbonAware.WebApi/README.md

```diff
@@ -79,6 +81,52 @@ EG
 }
 ```
 
+### GET emissions/bylocation
+
+This endpoint calculates the observed emission data by location for a specified time period
+
+location is a required parameter and is name of a data region for the given Cloud provider.
+If time period is not provided, it retrieves all the data until the current time.
```

Comment:
There is some nuance here.  `All available data` might be more accurate, though for WattTime that's still not technically accurate because it wont include historical data :)